tests/activity_log_fixtures.json

```diff
@@ -0,0 +1,25 @@
+{"records":[
+    {
+        "time":"2018-03-07T14:23:51.991Z",
+        "resourceId":"/SUBSCRIPTIONS/C088DC46-D692-42AD-A4B6-9A542D28AD2A/RESOURCEGROUPS/TESTEVENTHUBLOGS/PROVIDERS/MICROSOFT.WEB/SITES/SUMOAZUREAUDITAPP/CONFIG/APPSETTINGS",
+        "operationName":"MICROSOFT.WEB/SITES/CONFIG/WRITE",
+        "category":"Write","resultType":"Start","resultSignature":"Started.","durationMs":0,"callerIpAddress":"111.93.54.106","correlationId":"cdab1f54-3d65-4d32-91ce-829db00381ae",
+        "identity":{
+            "authorization":{
+                "scope":"/subscriptions/c088dc46-d692-42ad-a4b6-9a542d28ad2a/resourceGroups/TestEventHubLogs/providers/Microsoft.Web/sites/SumoAzureAuditApp/config/appsettings",
+                "action":"Microsoft.Web/sites/config/write",
+                "evidence":{"role":"Owner","roleAssignmentScope":"/subscriptions/c088dc46-d692-42ad-a4b6-9a542d28ad2a","roleAssignmentId":"5c6afc93f94942b38879b1ed60aca922","roleDefinitionId":"8e3af657a8ff443ca75c2fe8c4bcb635","principalId":"95fdefa0a7b847a3b969046862c9a979","principalType":"User"
+                }
+            },
+            "claims":{
+                "aud":"https://management.core.windows.net/",
+                "iss":"https://sts.windows.net/a39bedba-be8f-4c0f-bfe2-b8c7913501ea/","iat":"1520430057","nbf":"1520430057",
+                "exp":"1520434256",
+                "http://schemas.microsoft.com/claims/authnclassreference":"1","aio":"Y2NgYDgv1K/hPiFd+27SQn2lsKBIz8+rvPUk1y/1femdVL7390kA",
+                "http://schemas.microsoft.com/claims/authnmethodsreferences":"pwd",
+                "appid":"f64071b9-a79b-4655-9dad-3b3535e00b84","appidacr":"2","e_exp":"263099","http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname":"Pal","http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname":"Himanshu","ipaddr":"111.93.54.106","name":"Himanshu Pal","http://schemas.microsoft.com/identity/claims/objectidentifier":"95fdefa0-a7b8-47a3-b969-046862c9a979","puid":"10030000A77E725E","http://schemas.microsoft.com/identity/claims/scope":"user_impersonation","http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier":"mucf8YYRxalNePxQtx2t_Vp7WtIYsjrE68K5r0JCo9k","http://schemas.microsoft.com/identity/claims/tenantid":"a39bedba-be8f-4c0f-bfe2-b8c7913501ea","http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name":"himanshu@itsumologic.onmicrosoft.com","http://schemas.xmlsoap.org/ws/2005/05/identity/claims/upn":"himanshu@itsumologic.onmicrosoft.com","uti":"kINdgmA35kmIMsXXinoXAA","ver":"1.0"
```

Comment:
Is there anything sensitive (credit, key) here?

Reply:
not any keys but I guess I should remove my account name himanshu@itsumologic.onmicrosoft.com from there